### PR TITLE
Show menu entry when a manual or automatic workflow is active

### PR DESCRIPTION
**Pull request by justusdieckmann**

### CODE REVIEW

File: classes/local/manager/workflow_manager.php [hunk 1]
```diff
@@ -249,7 +249,7 @@ public static function activate_workflow($workflowid) {
      */
     private static function reset_manual_workflow_cache() {
```
Comment (PhMemmel) [inline classes/local/manager/workflow_manager.php:250]: I think it would be a good idea to change the name of this method as well?

Comment (justusdieckmann) [reply classes/local/manager/workflow_manager.php]: You're right of course, thanks!

**Comment by NinaHerrmann** [comment]
sorry i did not really think about it

**Comment by NinaHerrmann** [comment]
Nearly forgot about that one... Looks good thank you so much! :smiling_face_with_three_hearts:

**Pull request merged**